reports/models.py

```diff
@@ -0,0 +1,20 @@
+from sre_parse import CATEGORIES
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class Report(models.Model):
+    """
+    Name and file path of generated reports.
+    """
+
+    name = models.CharField(max_length=20)
+    directory = models.CharField(max_length=250)
```

Comment:
CharField may work, but why? Why not a FileField?

Reply:
I didn't know that type of field exists.
Addressed: https://github.com/jino-d/warehouse/pull/2/commits/3098a7fc8096d85b5ef421b9aaeb8f11e7a436d3